fakebook/src/app/service/user.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from '../model/user';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+  constructor(private http: HttpClient) {}
+
+  // test url
+  url = `https://2011-project2-fakebook.azurewebsites.net/api/`; // url of the api
```

Comment:
that's not the url of the API.
like i said, not maintainable... should be in environment.

Reply:
changed to use environment